src/fides/data/sample_project/fides.toml

```diff
@@ -22,6 +22,7 @@ root_password = "Testpassword1!"
 
 [execution]
 require_manual_request_approval = true
+masking_strict = true
```

Comment:
Make sure to roll this back, you can set this value in the `.env` file in the `fides` root directory instead
```
FIDES__EXECTION__MASKING_STRICT=true
```